CHANGELOG.md

```diff
@@ -5,11 +5,16 @@ It was first called Evopedia (and was using the file format of Evopedia).
 Then it was renamed Kiwix-html5 (and uses ZIM files), and then again was renamed to Kiwix-JS.
 
 ## Kiwix-JS v2.8.0
```

Comment:
I usually add the following version (2.9.0) at the top, with the TODOs

Reply:
@mossroy OK, but I thought you added that as a commit *after* the creation of the v2.80 tag, like last time (see screenshot). Here is the last [Prepare source code for version 2.8](https://github.com/kiwix/kiwix-js/commit/b4124bbaad13a5a2e286b7789397b1f3b0edd050). It would make for a cleaner release not to have the source code for the next release in it? But just let me know what you'd prefer!

![image](https://user-images.githubusercontent.com/4304337/86947502-c62c8f00-c143-11ea-8a79-79d116cfa8fb.png)

Reply:
You're probably right. So let's keep it like this